setup/update/sql/2.2.0-alpha1.sql

```diff
@@ -1,53 +1,81 @@
-  (@max_id + 1, "fr_FR", "Remboursée", "", "", "")
+insert into  `order_status_i18n` values
+  (@max_id + 1, "en_us", "refunded", "", "", ""),
+  (@max_id + 1, "fr_fr", "remboursée", "", "", "")
 ;
 
 -- new column in admin_log
 
-ALTER TABLE `admin_log` ADD `resource_id` INTEGER AFTER `resource` ;
+alter table `admin_log` add `resource_id` integer after `resource` ;
 
 -- new config
 
-SELECT @max_id := IFNULL(MAX(`id`),0) FROM `config`;
+select @max_id := ifnull(max(`id`),0) from `config`;
 
-INSERT INTO `config` (`id`, `name`, `value`, `secured`, `hidden`, `created_at`, `updated_at`) VALUES
-(@max_id + 1, 'customer_change_email', '0', 0, 0, NOW(), NOW()),
-(@max_id + 2, 'customer_confirm_email', '0', 0, 0, NOW(), NOW())
+insert into `config` (`id`, `name`, `value`, `secured`, `hidden`, `created_at`, `updated_at`) values
+(@max_id + 1, 'customer_change_email', '0', 0, 0, now(), now()),
+(@max_id + 2, 'customer_confirm_email', '0', 0, 0, now(), now())
 ;
 
-INSERT INTO `config_i18n` (`id`, `locale`, `title`, `description`, `chapo`, `postscriptum`) VALUES
-(@max_id + 1, 'en_US', 'Allow customers to change their email. 1 for yes, 0 for no', NULL, NULL, NULL),
-(@max_id + 1, 'fr_FR', 'Permettre aux clients de changer leur email. 1 pour oui, 0 pour non', NULL, NULL, NULL),
-(@max_id + 2, 'en_US', 'Ask the customers to confirm their email, 1 for yes, 0 for no', NULL, NULL, NULL),
-(@max_id + 2, 'fr_FR', 'Demander aux clients de confirmer leur email. 1 pour oui, 0 pour non', NULL, NULL, NULL)
+insert into `config_i18n` (`id`, `locale`, `title`, `description`, `chapo`, `postscriptum`) values
+(@max_id + 1, 'en_us', 'allow customers to change their email. 1 for yes, 0 for no', null, null, null),
+(@max_id + 1, 'fr_fr', 'permettre aux clients de changer leur email. 1 pour oui, 0 pour non', null, null, null),
+(@max_id + 2, 'en_us', 'ask the customers to confirm their email, 1 for yes, 0 for no', null, null, null),
+(@max_id + 2, 'fr_fr', 'demander aux clients de confirmer leur email. 1 pour oui, 0 pour non', null, null, null)
 ;
 
-SET FOREIGN_KEY_CHECKS = 1;
\ No newline at end of file
+-- country area table
+
+create table `country_area`
+(
+    `country_id` integer not null,
+    `area_id` integer not null,
+    `created_at` datetime,
+    `updated_at` datetime,
+    index `country_area_area_id_idx` (`area_id`),
+    index `fk_country_area_country_id_idx` (`country_id`),
+    constraint `fk_country_area_area_id`
+        foreign key (`area_id`)
+        references `area` (`id`)
+        on update restrict
+        on delete cascade,
+    constraint `fk_country_area_country_id`
+        foreign key (`country_id`)
+        references `country` (`id`)
+        on update restrict
+        on delete cascade
+) engine=innodb character set='utf8';
+
+-- Initialize the table with existing data
+INSERT INTO `country_area` (`country_id`, `area_id`, `created_at`, `updated_at`) select `id`, `area_id`, NOW(), NOW() FROM `country` WHERE area_id <> NULL
```

Comment:
you change all this file from Upper case to lower case and here you write your query in upper case...